dcos/mesos.py

```diff
@@ -40,8 +40,21 @@ def __init__(self):
         else:
             self._mesos_master_url = mesos_master_url
 
+    def get_dcos_url(self, path):
```

Comment:
we already loaded the config in the constructor, so this is using the already-loaded dcos_url, and failing if we can't find it.  We don't fail in the constructor because you're allowed to instantiate this class without dcos_url set, but you can't use this method.